plugins/pitchfork.py

```diff
@@ -3,14 +3,20 @@
 import textwrap
 
 from errbot import BotPlugin, botcmd
+from plugins.labhub import members_only
```

Comment:
we do not want to make everything dependant on labhub in order to prevent people from misusing all of the other commands.